tests/ui/resolve/prelude-order.rs

```diff
@@ -58,7 +58,7 @@ extern crate macro_helpers as _;
 /* lang and libs implicitly in scope */
 
 // tool/extern -> extern
-#[type_ns::inner] //~ ERROR could not find `inner` in `type_ns`
+#[type_ns::inner] //~ ERROR cannot find macro `inner` in crate `macro_helpers`
```

Comment:
```suggestion
#[type_ns::inner] //~ ERROR cannot find macro `inner` in crate `type_ns`
```
would be preferable, although I'm not sure.